docs/sources/style-guide/write-for-developers/index.md

```diff
@@ -177,18 +177,18 @@ ssh-rsa KEY_VALUE USERNAME [_FILENAME_]
 ```
 
 Use descriptive words and phrases when including placeholders, and avoid using X or XXX.
-In Markdown, in front of a placeholder, use an asterisk followed by a backtick. At the end of the placeholder, use a backtick followed by an asterisk. For example, prefer the following raw markdown:
+In Markdown, in front of a placeholder, use an asterisk followed by a backtick and an open bracket. Use a dash instead of an underscore in the placeholder name if necessary. At the end of the placeholder, use an open bracket followed by a backtick and an asterisk. For example, prefer the following raw markdown:
 
 ````
 ```
-The following text is a placeholder: *`A_PLACEHOLDER`*.
+The following text is a placeholder: *`<A-PLACEHOLDER>`*.
 ```
 ````
 
 It produces:
 
 ```
-The following text is a placeholder: *`A_PLACEHOLDER`*.
+The following text is a placeholder: *`<A-PLACEHOLDER>`*.
```

Comment:
This formatting doesn't make any sense to me.  Is this supposed to be bold and code formatted in the output?

Reply:
I believe so. When I updated this topic that was the existing style and I didn't change it.

Reply:
Didn't you write this topic?

Reply:
Did the formatting get reversed?  If it's supposed to be bold and code formatted I think the bolding (*) should be **_inside_** the code formatting (` `)?
```suggestion
The following text is a placeholder: `*<A-PLACEHOLDER>*`.
```